### PR TITLE
New Indonesian translations

### PR DESCRIPTION
**Pull request by peterp**
I had to translate some molo template strings, I just extracted them from site-packages.

### CODE REVIEW

File: gem/templates/profiles/viewprofile.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {% block content %}
     <div class="profile-container">
         <div class="title profile">
-            <h3>{% trans "Hello " %}{{ user }}</h3>
+            <h3>{% trans "Hello " %} {{ user }}</h3>
```
Comment (smn) [inline gem/templates/profiles/viewprofile.html]: The space already seems to be in the translation string :confused:

Comment (miltontony) [inline gem/templates/profiles/viewprofile.html]: yeah, lets remove the space from the translation. It should just be `{% trans "Hello" %}`

File: locale/id/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -181,94 +175,96 @@ msgid "Edit your Profile"
...
 
-#: gem/templates/profiles/register.html:26
+#: gem/templates/profiles/register.html:27
 msgid "already joined?"
 msgstr "Sudah bergabung?"
 
-#: gem/templates/profiles/viewprofile.html:6
-msgid "Hello "
+#: gem/templates/profiles/viewprofile.html:7
+#, fuzzy
```
Comment (miltontony) [inline locale/id/LC_MESSAGES/django.po]: Please delete this line - basically django is confirming that it did a partial match and found these 2 strings to still be the same. If you leave this in, `+#, fuzzy` it ignores this when the translation compiles.

**Comment by miltontony** [comment]
:+1:

**Pull request merged**